.coafile

```diff
@@ -47,3 +47,112 @@ ignore = vendor/**, resources/vendors/**, _site/**
 bears = YAMLLintBear
 files = **.yml
 ignore = vendor/**
+
+[future_linelength_projects]
```

Comment:
could be just `linelength`

Reply:
But this name specifies that these are for projects folder and not any other files. So I think that this name should be kept.

Reply:
well because of the underscore in the `_project` folder and also underscore being used as the word delimiter in names it was kinda getting fuzzy.

Reply:
But it is still understandable whereas if we name it linelength it would become a lot misleading.

Reply:
so devise a clever non-misleading name 😉